NextcloudTalk/UserStatusMessageSwiftUIView.swift

```diff
@@ -0,0 +1,369 @@
+                                    Text(status.icon ?? " ")
+                                    VStack(alignment: .leading) {
+                                        Text(status.message ?? "")
+                                            .foregroundColor(.primary)
+                                        Text(getPredefinedClearStatusText(clearAt: status.clearAt, clearAtTime: status.clearAtTime, clearAtType: status.clearAtType))
+                                            .font(.subheadline)
+                                            .foregroundColor(.secondary)
+                                            .lineLimit(1)
+                                    }
+                                    Spacer()
+                                }
+                            }
+                        }
+                    }
+                    Section {
+                        Button(action: {
+                            showClearAtAlert.toggle()
+                        }) {
+                            VStack(alignment: .leading) {
+                                Text(NSLocalizedString("Clear status message after", comment: ""))
+                                    .foregroundColor(.primary)
+                                Text(selectedClearAtString)
+                                    .font(.subheadline)
+                                    .foregroundColor(.secondary)
+                                    .lineLimit(1)
+                            }
+                        }
+                    }
+                }
+                if !textFieldIsFocused {
+                    VStack(spacing: 20) {
+                        VStack(spacing: 10) {
+                            NCButtonSwiftUI(title: NSLocalizedString("Clear status message",
+                                                                     comment: ""),
+                                            action: clearActiveUserStatus,
+                                            style: .tertiary, height: 40,
+                                            disabled: Binding.constant(selectedMessage.isEmpty))
+                            NCButtonSwiftUI(title: NSLocalizedString("Set status message", comment: ""),
+                                             action: setActiveUserStatus,
+                                            style: .primary, height: 40,
+                                            disabled: Binding.constant(selectedMessage.isEmpty))
+                        }
+                    }
+                    .padding(.bottom, 25)
+                }
+            }
+        }
+        .navigationBarTitle(Text(NSLocalizedString("Status message", comment: "")), displayMode: .inline)
+        .navigationBarHidden(false)
+        .onAppear {
+            getStatus()
+        }
+        .confirmationDialog("Clear status message after", isPresented: $showClearAtAlert, titleVisibility: .visible) {
+            Button("Don't clear") {
```

Comment:
Is this localized this way?

Reply:
Ja, swiftUI macht das automatisch

![Bildschirmfoto 2023-08-16 um 15 03 09](https://github.com/nextcloud/talk-ios/assets/106263486/2f59f384-f4b3-47ca-aa9b-efc4788e35df)